test/mocks/schemes/scheme.js

```diff
@@ -0,0 +1,7 @@
+const { SFI } = require('../../../app/constants/schemes')
+
+module.exports = {
+  schemeId: SFI,
+  name: 'Scheme name',
```

Comment:
Should this be changed to SFI's scheme name or?

Reply:
Changed.